libraries/Microsoft.Bot.Builder/Microsoft.Bot.Builder.csproj

```diff
@@ -51,6 +55,7 @@
 		<PackageReference Include="Microsoft.Bot.Connector" Condition=" '$(PackageVersion)' == '' " Version="4.0.0-local" />
 		<PackageReference Include="Microsoft.Bot.Connector" Condition=" '$(PackageVersion)' != '' " Version="$(PackageVersion)" />
 		<PackageReference Include="SourceLink.Create.CommandLine" Version="2.8.1" />
+		<PackageReference Include="StyleCop.Analyzers" Version="1.0.2" />
```

Comment:
:memo: This needs to use the latest 1.1.0 beta for C# 7 support.

```xml
<PackageReference Include="StyleCop.Analyzers" Version="1.1.0-beta008" PrivateAssets="all" />
```

Reply:
Ahhh. I went with "Stable". :)